src/modules/Support/html_client/mod_support_ticket.html.twig

```diff
@@ -4,199 +4,213 @@
                                     </div>
-                                    {% endif %}
-                                </header>
-                                <section>
-                                    <div class="well" id="message-{{message.id}}">{{ message.content|markdown }}</div>
-                                </section>
+                                </div>
                             </div>
                         </div>
-                        <hr/>
-                {% endfor %}
-
-                <div class="row-fluid">
-                    {% if ticket.status != 'closed' %}
-                    <div class="span3"></div>
-                    <div class="span9">
-                        <form method="post" action="" class="api_form" data-api-url="{{ 'api/client/support/ticket_reply'|link }}" data-api-reload="1">
-                            <input type="hidden" name="CSRFToken" value="{{ CSRFToken }}"/>
-                            <fieldset>
-                                <textarea name="content" cols="5" rows="10" class="span12" required="required" id="ticket-reply-text"></textarea>
-                                <button class="btn btn-primary btn-large" type="submit" id="submit-support-message" value="{{ 'Post'|trans }}" onclick="">{{ 'Post'|trans }}</button>
-                                <input type="hidden" name="id" value="{{ ticket.id }}">
-                            </fieldset>
-                        </form>
                     </div>
+                </div>
+            {% endfor %}
+        </div>
+        <div>
+            <div class="row">
+                <div class="col-md-12">
+                    {% if ticket.status != 'closed' %}
+                        <div class="card mt-3" id="reply-to">
+                            <div class="card-body">
+                                <form id="ticket-reply-form">
+                                    <input type="hidden" name="CSRFToken" value="{{ CSRFToken }}"/>
+                                    <input type="hidden" name="id" value="{{ ticket.id }}">
+                                    <textarea name="content" cols="5" rows="10" class="form-control" required="required"
+                                              id="ticket-reply-text"></textarea>
+                                    <button class="mt-2 btn btn-primary" type="submit" id="post-reply-btn"
+                                            value="{{ 'Post'|trans }}"
+                                            onclick="">{{ 'Post'|trans }}</button>
+                                </form>
+                            </div>
+                        </div>
                     {% elseif ticket.status == 'closed' %}
-                        <div class="alert alert-white text-center">{{ 'Ticket was closed and cannot be reopened.'|trans }}</div>
+                        <div class="mt-3 alert alert-secondary text-center">{{ 'Ticket was closed and cannot be reopened.'|trans }}</div>
```

Comment:
This doesn't respect the setting anymore.
![grafik](https://github.com/FOSSBilling/FOSSBilling/assets/1939311/d94b3455-14d9-433a-b80f-62c9604dae44)

Reply:
>anymore

It never did, but I've gone in and fixed it nonetheless.
https://github.com/FOSSBilling/FOSSBilling/blob/main/src/modules/Support/html_client/mod_support_ticket.html.twig#L159